modules/eks/privatelink.tf

```diff
@@ -0,0 +1,12 @@
+module "privatelink" {
+  count                    = var.privatelink.enabled ? 1 : 0
+  source                   = "./submodules/privatelink"
+  deploy_id                = var.deploy_id
+  region                   = var.region
+  vpc_endpoint_services    = var.privatelink.vpc_endpoint_services
```

Comment:
could be consolidate and just pass `privatelink = var.privatelink`

Reply:
done